freqtrade/persistence/models.py

```diff
@@ -518,16 +576,31 @@ def calc_close_trade_value(self, rate: Optional[float] = None,
         if rate is None and not self.close_rate:
             return 0.0
 
-        sell_trade = Decimal(self.amount) * Decimal(rate or self.close_rate)  # type: ignore
-        fees = sell_trade * Decimal(fee or self.fee_close)
-        return float(sell_trade - fees)
+        close_trade = Decimal(self.amount) * Decimal(rate or self.close_rate)  # type: ignore
+        fees = close_trade * Decimal(fee or self.fee_close)
+
+        # TODO: Need to set other conditions because sometimes self.open_date is not defined, but why would it ever not be set
+        try:
+            open = self.open_date.replace(tzinfo=None)
+            now = datetime.now()
+
+            # breakpoint()
+            interest = ((Decimal(self.interest_rate or 0) * Decimal(self.borrowed or 0)) *
+                        Decimal((now - open).total_seconds())/86400) or 0  # Interest/day * (seconds in trade)/(seconds per day)
+        except:
+            interest = 0
```

Comment:
you will for sure want to extract this into a seperate method.
First of all, to be able to easily test it - but more importantly - because it'll also be needed for the "current profit" calculation - otherwise things like "ROI" will be off by this - so you think you sold for 0.5% - but instead it's a -0.5% loss - as we didn't account for all fees upfront